langchain/src/tools/gmail/create_draft.ts

```diff
@@ -0,0 +1,65 @@
+import { GmailBaseTool, GmailBaseToolParams } from "./base.js";
```

Comment:
The python version had create_draft so we wanted to allow the same tools available there. I'm guessing creating a draft might be useful because you may not fully trust the LLM to write AND send an email for you so you just want it to create a draft for you that you can potentially edit later?

Up to you though, if you don't feel like this is necessary we can delete it.